notebooks/00_quick_start/lightgbm_tinycriteo.ipynb

```diff
@@ -340,14 +350,14 @@
     }
    ],
    "source": [
-    "nume_cols = ['I'+str(i) for i in range(1, 14)]\n",
-    "cate_cols = ['C'+str(i) for i in range(1, 27)]\n",
-    "label_col = 'Label'\n",
+    "nume_cols = [\"I\" + str(i) for i in range(1, 14)]\n",
+    "cate_cols = [\"C\" + str(i) for i in range(1, 27)]\n",
+    "label_col = \"Label\"\n",
     "\n",
-    "from tempfile import TemporaryDirectory\n",
+    "header = [label_col] + nume_cols + cate_cols\n",
     "with TemporaryDirectory() as tmp:\n",
```

Comment:
better (and more consistent with movielens) to handle directory cleanup within the data loader

Reply:
changed using @loomlike suggestions